src/main/java/Duke.java

```diff
@@ -1,10 +1,131 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
seems like this class is handling interaction with user, processing of user command, and managing the task objects, which is multiple layers of abstraction
suggesting grouping the diffrent uses together under separate classes